interchain-exchange/tutorial/00-tutorial.md

```diff
@@ -16,23 +16,42 @@ If you want to see the end result, please refer to the [example implementation](
 - Send IBC packets from one blockchain to another
 - Deal with timeouts and acknowledgements of IBC packets
 
+## Concepts good to know
+
+**Order Books**
+
+**Order books** are electronic list of *buy* and *sell* orders for specific financial instrument - For eg: cryptocurrencies.
+An order book lists the number of tokens being *bid* (buy) on or *offered* (sell) at each price point, or market depth. It also identifies the market participants behind the buy and sell orders. These lists help traders and also improve market transparency because they provide valuable trading information.
```

Comment:
```suggestion
**Order books** are electronic list of *buy* and *sell* orders for specific financial instrument - For eg: cryptocurrencies. An order book lists the number of tokens being *bid* (buy) on or *offered* (sell) at each price point, or market depth. It also identifies the market participants behind the buy and sell orders. These lists help traders and also improve market transparency because they provide valuable trading information.
```